app/views/dinosaurs/show.html.erb

```diff
@@ -20,6 +26,7 @@ User reviews:
             <% r.comments.each do |comment| %>
               <span class="comment-body"><%= comment.body %> </span>
               <span class="comment-user">(<%= comment.user.email %>) </span>
+              <br>
```

Comment:
This is okay for now, but when adding more styling, use CSS to create space between elements instead of `<br>` tags.